Sources/Rendering/Core/ImageResliceMapper/example/index.js

```diff
@@ -108,6 +122,12 @@ function setImage(im) {
   planeState.onModified(() => {
     slicePlane.setOrigin(planeState.getOrigin());
     slicePlane.setNormal(planeState.getNormal());
+    slicePolyDataSource.setCenter(planeState.getOrigin());
+    slicePolyDataSource.setDirection(planeState.getNormal());
+    if (mapper.getSlicePolyData()) {
+      slicePolyDataSource.update();
+      mapper.setSlicePolyData(slicePolyDataSource.getOutputData());
```

Comment:
shouldn't this work with connections: `setInputConnection(1, slicePolyDataSource.getOutputPort())`

Reply:
It doesn't work as is but yeah, that could prove useful.